internal/amos/amos.go

```diff
@@ -1946,234 +1956,159 @@ OneTen:
 // DUE TO UNDERFLOW. NZ<0 MEANS UNDERFLOW OCCURRED, BUT THE
 // CONDITION CABS(Z)<=2*SQRT(FNU+1) WAS VIOLATED AND THE
 // COMPUTATION MUST BE COMPLETED IN ANOTHER ROUTINE WITH N=N-ABS(NZ).
-func Zseri(ZR, ZI, FNU float64, KODE, N int, YR, YI []float64, NZ int, TOL, ELIM, ALIM float64) (
-	ZRout, ZIout, FNUout float64, KODEout, Nout int, YRout, YIout []float64, NZout int, TOLout, ELIMout, ALIMout float64) {
-	var AA, ACZ, AK, AK1I, AK1R, ARM, ASCLE, ATOL,
-		AZ, CKI, CKR, COEFI, COEFR, CONEI, CONER, CRSCR, CZI, CZR, DFNU,
-		FNUP, HZI, HZR, RAZ, RS, RTR1, RZI, RZR, S, SS, STI,
-		STR, S1I, S1R, S2I, S2R, ZEROI, ZEROR float64
-	var I, IB, IDUM, IFLAG, IL, K, L, M, NN, NW int
-	var WR, WI [3]float64
-	var tmp, s2 complex128
+func Zseri(z complex128, fnu float64, kode, n int, y []complex128, tol, elim, alim float64) (nz int) {
+	var AA, AK, ATOL,
+		DFNU, FNUP, RS, S, SS float64
+	var I, IB, IDUM, IFLAG, IL, L, M, NN, NW int
+	var w [2]complex128
 	var sin, cos float64
+	var first bool
 
-	CONER = 1.0
-	NZ = 0
-	AZ = cmplx.Abs(complex(ZR, ZI))
-	if AZ == 0.0E0 {
-		goto OneSixty
```

Comment:
Ah, it's [here](https://github.com/gonum/mathext/pull/18/files#diff-f4a974e1c19635f3a4bd07971f170767R1983)